components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/util/APIErrorResponseHandler.java

```diff
@@ -219,12 +219,12 @@ private static String getRealmInfo() {
         return "Basic realm=" + ServerConfiguration.getInstance().getFirstProperty("HostName");
     }
 
-    private static boolean parseRemoveRealmUser() {
+    private static boolean parseAddRealmUser() {
 
-        String removeRealmUser = IdentityUtil.getProperty(REMOVE_REALM_USER_CONFIG);
-        if (StringUtils.isNotBlank(removeRealmUser)) {
-            return Boolean.parseBoolean(removeRealmUser);
+        String addRealmUser = IdentityUtil.getProperty(ADD_REALM_USER_CONFIG);
+        if (StringUtils.isNotBlank(addRealmUser)) {
```

Comment:
why do we need this if check

Reply:
This can be removed as this is unnecessary.
Addressed in aeb943898ab3af9e6eae2ccbe613149b9494a689